docs/.vuepress/config.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
               '/components/button',
               '/components/card',
               '/components/catalog',
+              // '/components/datetimepicker', // Leave commented out; we do not want to publicly release this component yet
```

Comment:
@kaiarrowood @aanchalm01 @jillztom we are leaving this docs page hidden on purpose as we will not publicly release this component until the Vue 3 version.

Ping @adamdehaven with questions :)